src/Command/RollbackCommand.php

```diff
@@ -130,6 +134,14 @@ protected function executeMigrations(Arguments $args, ConsoleIo $io): ?int
         $force = (bool)$args->getOption('force');
         $dryRun = (bool)$args->getOption('dry-run');
 
+        $count = $args->getOption('count') !== null ? (int)$args->getOption('count') : null;
+        if ($count && $date) {
+            throw new LogicException('Can only use one of `--count` or `--date` options at a time.');
+        }
```

Comment:
Wouldn't `--count=0` misbehave here? Maybe we need a guard for `count >= 1`?

Reply:
I figure 0 is the same as all.
Since it would be a no op setting.

Reply:
I am happy to protect users from themselves here.
Will add the check.